src/main/java/stashpullrequestbuilder/stashpullrequestbuilder/StashRepository.java

```diff
@@ -60,19 +61,17 @@
   private static final Pattern ADDITIONAL_PARAMETER_REGEX_PATTERN =
       Pattern.compile(ADDITIONAL_PARAMETER_REGEX);
 
-  private AbstractProject<?, ?> job;
+  private Job<?, ?> job;
   private StashBuildTrigger trigger;
   private StashApiClient client;
 
-  public StashRepository(@Nonnull AbstractProject<?, ?> job, @Nonnull StashBuildTrigger trigger) {
+  public StashRepository(@Nonnull Job<?, ?> job, @Nonnull StashBuildTrigger trigger) {
     this(job, trigger, null);
   }
 
   // For unit tests only
   StashRepository(
-      @Nonnull AbstractProject<?, ?> job,
-      @Nonnull StashBuildTrigger trigger,
-      StashApiClient client) {
+      @Nonnull Job<?, ?> job, @Nonnull StashBuildTrigger trigger, StashApiClient client) {
```

Comment:
minor: can you leave the line breaks as they were before?

Reply:
No. The code is formatted automatically. I tried adding line breaks, but `mvn package` removes them.

Reply:
Sorry, I forgot we already have the fmt-maven-plugin enabled.

I'm surprised it's joining already broken lines in this context, but it's not a big issue to me